app/src/ingest_policy_pdfs.py

```diff
@@ -45,20 +59,17 @@ def parse_pdf_and_add_to_db(
     # Match header in BEM manual
     header_pattern = r"(BEM\s\d*\s+\d+\sof\s\d+\s+\w.*)"
     text_split_by_header = re.split(header_pattern, contents)
-    current_title = ""
     body_content = ""
     start_new_section = True
     for text_contents in text_split_by_header:
         is_header, contents, start_new_section = get_header_and_is_current_section(
             text_contents, start_new_section
         )
         # Check if we need to start a new section
-        if is_header and start_new_section and body_content != "":
-            current_title = contents
-        else:
+        if not (is_header and start_new_section and body_content != ""):
```

Comment:
This is easier to interpret for me:
```suggestion
        if not is_header or not start_new_section or not body_content:
```